src/main/java/Event.java

```diff
@@ -0,0 +1,13 @@
+public class Event extends Task{
```

Comment:
Would adding a white space after "Task" be better?

Reply:
Agreed with @seanlaiys, adding a white space after "Task" would be following coding standard.